Client/Client.Qt/Widgets/TextEdit.hpp

```diff
@@ -47,16 +47,16 @@ public slots:
     void textChanged();
 
 private slots:
-    void styleButtonClick(const QString& symbolStart, const QString& symbolEnd);
+    void styleButtonClick(const QString& symbolStart, const QString& symbolEnd) const;
 
 private:
-    void delSymbolsInSelection(QString& text, int& start, int& end, int symbolSize);
-    void delSymbolsOutSelection(QString& text, int& start, int& end, int symbolSize);
-    void insertSymbolsInSelection(QTextCursor& cursor, int& start, int& end, int symbolSize, const QString& symbolStart,
-                                  const QString& symbolEnd);
-    void selectText(QTextCursor& cursor, int start, int end);
+    void delSymbolsInSelection(QString& text, int start, int end, int symbolSize) const;
```

Comment:
Should be tested I suppose, because the author of this code made '&' with int with a reason, maybe